models/public/yolo-v3-tiny-tf/yolo-v3-tiny-tf.md

```diff
@@ -0,0 +1,136 @@
+# yolo-v3-tiny-tf
+
+## Use Case and High-Level Description
+
+YOLO v3 Tiny is a real-time object detection model implemented with Keras\* from this [repository](https://github.com/david8862/keras-YOLOv3-model-set) and converted to TensorFlow\* framework. This model was pretrained on COCO\* dataset with 80 classes.
+
+## Conversion
+
+## Conversion
+
+1. Download or clone the official [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
```

Comment:
I don't see how this repository can be considered "official" in any way. I think it would be better to use "original".